grp/classic.gi

```diff
@@ -1917,6 +1917,122 @@ InstallMethod( Omega,
     { filt, e, d, R } -> OmegaCons( filt, e, d, Size( R ) ) );
 
 
+#############################################################################
+##
+#F  WallForm( <form>, <m>, <fld> ) . . . . . . . . . . . compute the wall of <m>
```

Comment:
Does it compute "the wall"? Or the "Wall form of <m> with respect to <form>".
```suggestion
#F  WallForm( <form>, <m>, <fld> ) . . .  compute Wall form of <m> wrt <form>
```

Or maybe simplify your life and just drop this here
```suggestion
#F  WallForm( <form>, <m>, <fld> )
```